src/pyqasm/maps/decomposition_rules.py

```diff
@@ -96,6 +96,67 @@ class AppliedQubit(Enum):
     },
 }
 
+ROTATIONAL_LOOKUP_RULES = {
+    BasisSet.CLIFFORD_T: {
+        "rz": {
+            CONSTANTS_MAP["pi"]: [
+                {"gate": "s"},
```

Comment:
Is it necessary to add a "gate" key in this data structure? What about defining the list something like - 
```python
...
"rx": {
            CONSTANTS_MAP["pi"]: [
                "h",
                "s",
                "s",
                "h",
            ]
...
```
 I think it should simplify implementation a little bit.